packages/react/src/button/Button.tsx

```diff
@@ -0,0 +1,77 @@
+'use client';
+import * as React from 'react';
+import { useButton } from '../use-button/useButton';
+import { useRenderElement } from '../utils/useRenderElement';
+import type { BaseUIComponentProps } from '../utils/types';
+
+/**
+ * A button component that can be used to trigger actions.
+ * Renders a `<button>` element.
+ *
+ * Documentation: [Base UI Button](https://base-ui.com/react/components/button)
+ */
+export const Button = React.forwardRef(function Button(
+  componentProps: Button.Props,
+  forwardedRef: React.ForwardedRef<HTMLButtonElement>,
+) {
+  const {
+    render,
+    className,
+    disabled: disabledProp = false,
+    nativeButton = true,
```

Comment:
I thought `native` would be a better prop name as it's already applied on the `Button` component, but as we already have the prop in other components, it's probably better to use the same name.

Reply:
Yeah the `NativeButtonProps` type is reused everywhere, including here now.